configure-c8y.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+
+if [ `id -u` != 0 ] ; 
+  then printf "Please run as sudo or root!\n"
+  exit
+fi
+printf "\n*********************************************"
+printf "\nThis script lets you setup a thinEdge connection to cumulocity IoT.\nWe assume that the installation was successful and all the necessary services are running.
+Just follow the instructions....."
+printf "\n*********************************************"
+
+printf "\n\n"
+read  -p 'Enter the Cumulocity tenant url e.g. [your-tenant.cumulocity.com]: ' tenant_url
+tenant_url=${tenant_url:-your-tenant.cumulocity.com}
+tedge config set c8y.url $tenant_url
+
+printf "\n\n"
+read -p 'Enter the path to your root certificates. On most Linux systems that is [/etc/ssl/certs]: ' path_root_cert
+path_root_cert=${path_root_cert:-/etc/ssl/certs}
+tedge config set c8y.root.cert.path $path_root_cert
+
+printf "\n\n"
+read -p 'Enter your device-id [my-device]: ' device_id
+device_id=${device_id:-my-device}
+{
+    tedge cert create --device-id $device_id
+} ||
+printf "\n\n"
+read -r -p "Would you like to remove the old cert and create a new one? (y)Yes/(n)No: " response
```

Comment:
Prompting the user for a yes/no input should use a similar convention to other common linux commands (like `apt-get`).

For example:

```
read -r -p "Would you like to remove the old cert and create a new one? [y/N]"
```
Where the uppercase `N` means that it is the default value if the user just presses `<Enter>`. The default value always depends on the use-case, but using the uppercase convention feels more linix-like.

Examples can be found [here](https://stackoverflow.com/questions/3231804/in-bash-how-to-add-are-you-sure-y-n-to-any-command-or-alias)